generator/chains/chain100_3.json

```diff
@@ -1,129 +0,0 @@
-
-{
-    "serviceID": 1, 
-    "chains": [
```

Comment:
After checking this old format again for specifying service chains, I realized that with our new format we are removing aspects of the old description format. We are removing the possibility to specify latency requirements per service chains. Good to be aware of this... I may need to bring this back in future versions.

Reply:
The code didn't really use this latency here. If you check the old parse method:
https://github.com/EricssonResearch/cloud-native-app-simulator/pull/21/files#diff-3d5817372b698c46a9877bbf482b1b4929a589d723339cc1b7bd84750bc1eed5L130
We parse it but when we are going to retrun the parsed element, we didn't return latency. 

On the other hand, with the current format, we do have the latency between multiple clusters. If we are going to add another latency parameter between the services in a chain, I think we can add it inside the called services section. We also should consider the point that there is no such feature in K8S.

Reply:
We decided to go with leave it for another issue for latency between microservices